core/config/config.go

```diff
@@ -164,6 +180,26 @@ var (
 		Required: true,
 		Usage:    "Load configuration from `FILE`",
 	}
+	EnvironmentFlag = cli.StringFlag{
+		Name:     "environment",
+		Required: true,
+		Usage:    "Set the environment (production or development)",
+	}
+	EthRpcUrlFlag = cli.StringFlag{
+		Name:     "eth-rpc-url",
+		Required: true,
+		Usage:    "Ethereum RPC URL",
+	}
+	EthWsUrlFlag = cli.StringFlag{
+		Name:     "eth-ws-url",
+		Required: true,
+		Usage:    "Ethereum WS URL",
+	}
+	AggregatorServerIpPortAddressFlag = cli.StringFlag{
```

Comment:
I know this was the old variable name, but don't like it anymore. Can we separate into two separate flags, `ListenHost` and `ListenPort`? This makes it more obvious that ip can be a hostname, plus feel like separating them is more canonical from what I've seen.

Also you can make the default ListenHost 0.0.0.0

Reply:
So which one of them should I pass in `config.AggregatorServerIpPortAddr` ? If you want both to be passed then the config struct needs to be refactored which I think I could do in a different PR to reduce complexity. (as I would have to make changes in aggregator.go as well)